### ISSUE TITLE
Checkpoints

### ISSUE THREAD

**Issue by mthorwald** [pgmikhael/protgps#5]
Can you tell me which checkpoint does what? Are these values obtained after running the sequences AUC-ROC? 
7c4853cd22080b250ef89af2a1b25102epoch=3.ckpt
32bf44b16a4e770a674896b81dfb3729epoch=26.ckpt

Thanks

**Comment by pgmikhael** [comment]
Hi,

The `7c4853cd22080b250ef89af2a1b25102epoch=3.ckpt` is trained on a dataset with sequence-similarity split. So, the test data is dissimilar from the training set. The `32bf44b16a4e770a674896b81dfb3729epoch=26.ckpt` checkpoint is trained on a random assignment of protein sequence to train, dev, or test set.

**Comment by mthorwald** [comment]
Thanks, but I'm not a bioinformaticist. Which should be used if i want to test how a specific protein sequence responds- I am looking at a protein known to bind lncRNA. What are the output values, AUC-ROC?
